src/profile/__snapshots__/ProfilePage.test.jsx.snap

```diff
@@ -1238,7 +1209,7 @@ exports[`<ProfilePage /> Renders correctly in various states while saving an edi
     className="profile-page-bg-banner bg-primary d-none d-md-block p-relative"
   />
   <div
-    className="container-fluid"
+    className="container"
```

Comment:
Are these changes to container instead of container-fluid desired?

Reply:
Yes... the app had a max-width before, so this preserves that.  I mean, I could put it back to fluid, but it makes it super wide.  I don't feel strongly; was just trying to keep it 'the same'